### PR TITLE
Apply perlcritic rule for useless interpolation of literal string

### PR DESCRIPTION
**Pull request by b10n1k**
This is propably the biggest load of files which should adapt to the rule which force single quotes when string doesnt need interpolation.
Includes also some minor changes for *Hash key with quotes* but not all of them.

https://progress.opensuse.org/issues/138416

### CODE REVIEW

**Review by baierjan — APPROVED** [review]

File: lib/OpenQA/Client/Archive.pm [hunk 1]
```diff
@@ -193,7 +193,7 @@ sub _progress_monitior ($ua, $tx) {
             $last_updated = time;
             if ($progress < $current) {
                 $progress = $current;
-                print("\rDownloading $filename: ", $size == $len ? 100 : $progress . "%");
+                print('\rDownloading $filename: ', $size == $len ? 100 : $progress . '%');
```
Comment (perlpunk) [inline lib/OpenQA/Client/Archive.pm:196]: This should be left unchanged.

**Review by Martchus — CHANGES_REQUESTED** [review]
Looks good except for the one occurrence mentioned by @perlpunk.

**Comment by b10n1k** [comment]
> ## [Codecov](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst) Report
> Attention: `14 lines` in your changes are missing coverage. Please review.
> 
> > Comparison is base [(`34cd7cf`)](https://app.codecov.io/gh/os-autoinst/openQA/commit/34cd7cff48fa6ce7aab822d4d37f3557916eceb1?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst) 98.37% compared to head [(`a63f08e`)](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst) 98.37%.
> 
> [Files](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)	Patch %	Lines
> [lib/OpenQA/WebAPI/Description.pm](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst#diff-bGliL09wZW5RQS9XZWJBUEkvRGVzY3JpcHRpb24ucG0=)	0.00%	[3 Missing ⚠️ ](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)
> [lib/OpenQA/WebAPI/Auth/OpenID.pm](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst#diff-bGliL09wZW5RQS9XZWJBUEkvQXV0aC9PcGVuSUQucG0=)	0.00%	[2 Missing ⚠️ ](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)
> [lib/OpenQA/WebAPI/Controller/Admin/JobGroup.pm](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst#diff-bGliL09wZW5RQS9XZWJBUEkvQ29udHJvbGxlci9BZG1pbi9Kb2JHcm91cC5wbQ==)	0.00%	[2 Missing ⚠️ ](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)
> [lib/OpenQA/Script/CloneJob.pm](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst#diff-bGliL09wZW5RQS9TY3JpcHQvQ2xvbmVKb2IucG0=)	50.00%	[1 Missing ⚠️ ](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)
> [lib/OpenQA/Shared/Controller/Session.pm](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst#diff-bGliL09wZW5RQS9TaGFyZWQvQ29udHJvbGxlci9TZXNzaW9uLnBt)	66.66%	[1 Missing ⚠️ ](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)
> [lib/OpenQA/WebAPI/Controller/API/V1/Worker.pm](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst#diff-bGliL09wZW5RQS9XZWJBUEkvQ29udHJvbGxlci9BUEkvVjEvV29ya2VyLnBt)	80.00%	[1 Missing ⚠️ ](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)
> [lib/OpenQA/WebAPI/Controller/Admin/AuditLog.pm](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst#diff-bGliL09wZW5RQS9XZWJBUEkvQ29udHJvbGxlci9BZG1pbi9BdWRpdExvZy5wbQ==)	0.00%	[1 Missing ⚠️ ](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)
> [lib/OpenQA/WebAPI/Controller/Admin/Influxdb.pm](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst#diff-bGliL09wZW5RQS9XZWJBUEkvQ29udHJvbGxlci9BZG1pbi9JbmZsdXhkYi5wbQ==)	75.00%	[1 Missing ⚠️ ](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)
> [lib/OpenQA/WebAPI/Controller/File.pm](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst#diff-bGliL09wZW5RQS9XZWJBUEkvQ29udHJvbGxlci9GaWxlLnBt)	66.66%	[1 Missing ⚠️ ](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)
> [lib/OpenQA/WebAPI/Controller/Running.pm](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst#diff-bGliL09wZW5RQS9XZWJBUEkvQ29udHJvbGxlci9SdW5uaW5nLnBt)	50.00%	[1 Missing ⚠️ ](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst)
> Additional details and impacted files
> [☔ View full report in Codecov by Sentry](https://app.codecov.io/gh/os-autoinst/openQA/pull/5448?src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst). 📢 Have feedback on the report? [Share it here](https://about.codecov.io/codecov-pr-comment-feedback/?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=os-autoinst).

What should i do about those?

**Comment by Martchus** [comment]
I guess we can make an exception for "92.63% of diff hit (target 100.00%)" here. The changed lines look simple enough.

**Review by Martchus — APPROVED** [review]

**Review by okurz — APPROVED** [review]

**Comment by okurz** [comment]
handling the conflict resolution and merge manually

**Pull request merged**